configuration/globalproperties/clinical_gp.xml

```diff
@@ -0,0 +1,115 @@
+<config>
+  <globalProperties>
+    <globalProperty>
+      <property>emrapi.sqlSearch.activePatients</property>
+      <value>select distinct
```

Comment:
@rubailly why is this file added? Does it differ from the Haiti one?

Reply:
[Done](https://github.com/mekomsolutions/openmrs-config-easydoc/pull/4/commits/be61da316052fc30be7d24f34667c66cbd27b74c)